services/ui-src/src/utils/auth/userContext.tsx

```diff
@@ -1,22 +1,20 @@
 import { createContext } from "react";
-import { CognitoUser } from "@aws-amplify/auth";
-
-export interface CustomCognitoUser extends CognitoUser {
-  role: string;
+export interface MCRUser {
+  email: string;
+  given_name: string;
+  family_name: string;
+  userRole: string | undefined;
```

Comment:
Prefer `userRole?: string` to match our other optional use below (`state?`).

Also, I don't think userRole should ever not be set. Maybe when we construct the `MCRUser` object if `userRole` is not present in IDM then we present some error page to the user asking them to contact the help desk for an issue with their role? I think that is work for later though and not part of this PR.